src/Microsoft.AspNet.SignalR.JS/jquery.signalR.transports.foreverFrame.js

```diff
@@ -69,6 +69,14 @@
         iframeClearThreshold: 50,
 
         start: function (connection, onSuccess, onFailed) {
+            if (connection.accessToken) {
+                if (onFailed) {
+                    connection.log("Forever Frame does not support connections that require a Bearer token to connect, such as the Azure SignalR Service.");
```

Comment:
Can this be a resource string like `jsonpNotSupportedWithAccessToken`?

Reply:
I don't think it's as easy here since there are no existing resources in this file. Also I'm not sure we'll be localizing these messages anyway.

Reply:
Also log messages aren't generally localized